content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -231,187 +235,187 @@ and in the logs to monitor the state of the auditing subsystem.
 
 ## Setup for multiple API servers
 
 If you're extending the Kubernetes API with the [aggregation
 layer](/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/),
-you can also set up audit logging for the aggregated apiserver. To do this,
+you can also set up audit logging for the aggregated API server. To do this,
 pass the configuration options in the same format as described above to the
-aggregated apiserver and set up the log ingesting pipeline to pick up audit
-logs. Different apiservers can have different audit configurations and
+aggregated API server and set up the log ingesting pipeline to pick up audit
+logs. Different API servers can have different audit configurations and
 different audit policies.
 
-## Log Collector Examples
+## Log collector examples
 
 ### Use fluentd to collect and distribute audit events from log file
 
 [Fluentd](https://www.fluentd.org/) is an open source data collector for unified logging layer.
 In this example, we will use fluentd to split audit events by different namespaces.
 
-{{< note >}}
-The `fluent-plugin-forest` and `fluent-plugin-rewrite-tag-filter` are plugins for fluentd.
-You can get details about plugin installation from
-[fluentd plugin-management](https://docs.fluentd.org/v1.0/articles/plugin-management).
-{{< /note >}}
-
-1. Install [`fluentd`](https://docs.fluentd.org/v1.0/articles/quickstart#step-1:-installing-fluentd),
-   `fluent-plugin-forest` and `fluent-plugin-rewrite-tag-filter` in the kube-apiserver node
-
-1. Create a config file for fluentd
 
-    ```
-    cat <<'EOF' > /etc/fluentd/config
-    # fluentd conf runs in the same host with kube-apiserver
-    <source>
-        @type tail
-        # audit log path of kube-apiserver
-        path /var/log/kube-audit
-        pos_file /var/log/audit.pos
-        format json
-        time_key time
-        time_format %Y-%m-%dT%H:%M:%S.%N%z
-        tag audit
-    </source>
-
-    <filter audit>
-        #https://github.com/fluent/fluent-plugin-rewrite-tag-filter/issues/13
-        @type record_transformer
-        enable_ruby
-        <record>
-         namespace ${record["objectRef"].nil? ? "none":(record["objectRef"]["namespace"].nil? ? "none":record["objectRef"]["namespace"])}
-        </record>
-    </filter>
-
-    <match audit>
-        # route audit according to namespace element in context
-        @type rewrite_tag_filter
-        <rule>
-            key namespace
-            pattern /^(.+)/
-            tag ${tag}.$1
-        </rule>
-    </match>
-
-    <filter audit.**>
+1. [Install `fluentd`](https://docs.fluentd.org/v1.0/articles/quickstart#step-1:-installing-fluentd),
+   adding the plugins `fluent-plugin-forest` and `fluent-plugin-rewrite-tag-filter`, onto the each
+   host that runs the kube-apiserver.
+
+   {{< note >}}
+   The `fluent-plugin-forest` and `fluent-plugin-rewrite-tag-filter` are plugins for fluentd.
+   You can get details about plugin installation from
```

Comment:
Not sure if the note is required. The reader can read about the plugins in the fluentd documentation.